api/scaling.go

```diff
@@ -92,6 +92,7 @@ type TaskGroupScaleStatus struct {
 }
 
 type ScalingEvent struct {
+	Count       *int64
```

Comment:
In the returned data, when would `Count` or `EvalID` be nil?  Maybe some documentation would help clarify stuff here?

Reply:
when an "informational" scaling event happened, where Count was submitted as null for the propose of registering information from the autoscaler.